content-src/components/TopSites/_TopSites.scss

```diff
@@ -406,8 +406,6 @@ $hover-transition-duration: 150ms;
       font-size: 13px;
       margin-top: 4px;
       cursor: pointer;
-      border: 0;
-      color: var(--newtab-link-primary-color);
 
       &:hover {
         text-decoration: underline;
```

Comment:
Ah yup, looks like there's no more `_TopSites.scss` changes in the PR right now, but this `:hover` style should be redundant with the one in `_A11yLinkButton.scss`